skills/csharp/experimental/eventskill/Startup.cs

```diff
@@ -103,11 +97,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddHostedService<QueuedHostedService>();
 
             // Configure responses
-            services.AddSingleton(sp => new ResponseManager(
-                settings.CognitiveModels.Select(l => l.Key).ToArray(),
-                new MainResponses(),
-                new SharedResponses(),
-                new FindEventsResponses()));
+            services.AddSingleton(LocaleTemplateManagerWrapper.CreateLocaleTemplateManager("en-us"));
```

Comment:
Actually it has 6 locales except for FindEventsResponses

Reply:
fixed.